fairing/strategies/basic.py

```diff
@@ -1,19 +1,27 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
 class BasicTrainingStrategy(object):
+
   def __init__(self):
     self.runs = 1
     self.arch = None
     self.backend = None
+    logger.setLevel(logging.DEBUG)
 
   def add_training(self, svc, repository, image_name, image_tag, volumes, volume_mounts):
    return self.arch.add_jobs(svc, self.runs, repository, image_name, image_tag, volumes, volume_mounts), None
 
   def get_params(self):
     return {}
   
-  def exec_user_code(self, user_object):
+  def exec_user_code(self, user_object, *args, **kwargs):
+    logger.warn("execing user code with args {}", str(*args))
```

Comment:
Did you forget to remove this?
If not shoudln't this be `debug` and not `warn`?

Reply:
will change to debug

Reply:
lowered to debug.